### PR TITLE
Update 05_pet_breeds.ipynb

### PR DESCRIPTION
**Pull request by zhaoshaojun**
There should be an extra negative sign before sm_acts[idx, targ] to make it the correct loss for the following reason:

Take targ[0] as an example. Because targ[0] is 0, when sm_acts[0,0] is high, the prediction shows it is very much a 0, so the loss should be low. Therefore, the higher sm_acts[0,0] is, the lower the loss is. Therefore, an extra negative sign is needed before sm_acts[idx, targ] to make it the correct loss. Same when targ[i] is 1.

Behind the scene, CrossEntropyLoss has an extra log on top of -sm_acts[idx, targ] (take the log, then negate, of course). This can be verified by checking sm_acts[idx, targ].log()*(-1) == nn.CrossEntropyLoss(reduction='none')(acts, targ). The results should be true.

### CODE REVIEW

**Comment by jph00** [comment]
Many thanks
